libfwupd/fwupd-enums.h

```diff
@@ -697,6 +697,26 @@ typedef guint64 FwupdDeviceProblem;
  * Since: 1.7.5
  */
 #define FWUPD_RELEASE_FLAG_IS_COMMUNITY (1u << 7)
+/**
+ * FWUPD_RELEASE_FLAG_UPDATE_ACTION_DO_NOT_UNPLUG_POWER:
+ *
+ * Show the user a message not to unplug the machine from the AC power, e.g.
+ *
+ * "Do not turn off your computer or remove the AC adaptor while the update is in progress."
+ *
+ * Since: 1.8.6
+ */
+#define FWUPD_RELEASE_FLAG_UPDATE_ACTION_DO_NOT_UNPLUG_POWER (1u << 8)
+/**
+ * FWUPD_RELEASE_FLAG_UPDATE_ACTION_REMOVE_REPLUG:
+ *
+ * Show the user a message to replug the USB cable, e.g.
+ *
+ * "The update will continue when the device USB cable is unplugged and then re-inserted."
```

Comment:
This is from Dell dock right? I don't think it technically needs to be reinserted. The update happens whether the computer returned or not.

Reply:
This is actually for CCGX docks like the HP one and various other reference designs from Synaptics.

Reply:
If that's the case then isn't there two different statements needed since Dell and HP ones work slightly differently?